src/Table/Footer.jsx

```diff
@@ -120,10 +129,13 @@ Footer.propTypes = {
   onPageChange: PropTypes.func,
   numColumns: PropTypes.number.isRequired,
   numPages: PropTypes.number.isRequired,
+  showLastPage: PropTypes.bool,
+  isLoading: PropTypes.bool,
 };
 
 Footer.defaultProps = {
   onPageChange: () => {},
+  showLastPage: true,
```

Comment:
we did, but carl conceded that it's ok to not show the last page since it would result in a ton of fetches (you'd basically be paging through all the data)